init/init.go

```diff
@@ -24,18 +23,22 @@ func userPrompt() (string, string, string, string, string, string) {
 }
 
 func writeConfig(fs afero.Fs, config *v1.Config) error {
-	json, e := json.MarshalIndent(config, "", "  ")
-	if e != nil {
-		return errs.WrapInternal(e, "unable to marshal json")
+	yaml, err := yaml.Marshal(config)
+
+	if err != nil {
+		return errs.WrapInternal(err, "unable to marshal yaml")
 	}
-	configFile, e := fs.Create("fogg.json")
-	if e != nil {
-		return errs.WrapInternal(e, "unable to create config file fogg.json")
+
+	yamlConfigFile, err := fs.Create("fogg.yml")
+	if err != nil {
+		return errs.WrapInternal(err, "unable to create config file fogg.yml")
 	}
-	_, e3 := configFile.Write(json)
-	return e3
+	_, err = yamlConfigFile.Write(yaml)
+
```

Comment:
rm extra whitespace

Reply:
Does the team have a set of general best practices for go or just in general?

Reply:
we don't yet (and this is just a matter of style so opinions differ)